### PR TITLE
update docs with get forward plans endpoint

### PR DESCRIPTION
**Pull request by shaunganley**

### CODE REVIEW

File: docs/api-documentation/V1.4/index.md [hunk 1]
```diff
@@ -1364,6 +1364,24 @@ Query params:
...
+{: .govuk-body}
+
+<ol class="govuk-list govuk-list--bullet">
+  <li><strong>forward_plan_status</strong>: The forward plan status i.e. raised, closed, cancelled</li>
+  <li><strong>sort_column</strong>: The property of the foward plan to order results by</li>
+  <li><strong>sort_direction</strong>: Ascending/descending</li>
+  <li><strong>proposed_start_date</strong>: Date range filtering based on the proposed_start_date property</li>
+  <li><strong> proposed_end_date </strong>: Date range filtering based on the proposed_end_date property</li>
+  <li><strong>swa_code</strong>: Optional parameter to be used by contractors only. Used to provide the swa code of the promoter the contractor is working on behalf of</li>
+  <li><strong>query</strong>: The work reference number associated with the permit (partial match)</li>
```
Comment (pwright08) [inline docs/api-documentation/V1.4/index.md]: Should this be 
```suggestion
  <li><strong>query</strong>: The work reference number associated with the forward plan (partial match)</li>
```
?

File: docs/api-documentation/V1.4/index.md [hunk 2]
```diff
@@ -1364,6 +1364,24 @@ Query params:
...
+{: .govuk-heading-s}
+
+<code>GET /forward-plans</code>
+
+Query params:
+{: .govuk-body}
+
+<ol class="govuk-list govuk-list--bullet">
+  <li><strong>forward_plan_status</strong>: The forward plan status i.e. raised, closed, cancelled</li>
+  <li><strong>sort_column</strong>: The property of the foward plan to order results by</li>
```
Comment (pwright08) [inline docs/api-documentation/V1.4/index.md]:
```suggestion
  <li><strong>sort_column</strong>: The property of the forward plan to order results by</li>
```

File: docs/api-documentation/V1.4/index.md [hunk 3]
```diff
@@ -1364,6 +1364,24 @@ Query params:
...
+  <li><strong>sort_direction</strong>: Ascending/descending</li>
+  <li><strong>proposed_start_date</strong>: Date range filtering based on the proposed_start_date property</li>
+  <li><strong> proposed_end_date </strong>: Date range filtering based on the proposed_end_date property</li>
```
Comment (pwright08) [inline docs/api-documentation/V1.4/index.md]:
```suggestion
  <li><strong> proposed_end_date</strong>: Date range filtering based on the proposed_end_date property</li>
```

File: docs/api-documentation/V1.4/index.md [hunk 4]
```diff
@@ -1364,6 +1364,24 @@ Query params:
...
+
+Query params:
+{: .govuk-body}
+
+<ol class="govuk-list govuk-list--bullet">
+  <li><strong>forward_plan_status</strong>: The forward plan status i.e. raised, closed, cancelled</li>
+  <li><strong>sort_column</strong>: The property of the forward plan to order results by</li>
+  <li><strong>sort_direction</strong>: Ascending/descending</li>
+  <li><strong>proposed_start_date</strong>: Date range filtering based on the proposed_start_date property</li>
+  <li><strong>proposed_end_date</strong>: Date range filtering based on the proposed_end_date property</li>
```
Comment (pwright08) [inline docs/api-documentation/V1.4/index.md]: Maybe one for the client PR as well - but seems strange that the DB columns are start/end date, but the request fields are **proposed** start/end date

Comment (Kellym-Kainos) [reply docs/api-documentation/V1.4/index.md]: 🙌

**Pull request merged**